snaps_k8s/playbooks/k8/multus_node_bin.yaml

```diff
@@ -40,5 +40,13 @@
       dest: /opt/cni/bin/
       remote_src: yes
 
+  - name: Clone multus binary
+    shell: git clone https://github.com/AricentK8SRes/multus.git
```

Comment:
Is it possible find a multus binary from a standard download location?

Reply:
Hi @spisarski Steven,

We have tried using different multus binary present on multus github but some issues related to networking are being occurred.
So as of now, we can keep this complied binary and keep debugging parallelly.

Please let me know your view on this.

Regards,
Chirag Gupta

Reply:
That should be fine for now. Can you leverage the ansible "git" task rather than the "shell" task here? I will create a ticket on the backlog to address this binary.

Reply:
we are working on this.